databases/week1/week1.sql

```diff
@@ -0,0 +1,48 @@
+show DATABASES;
+use my_database_week1;
+SELECT * 
+from task
+SELECT COUNT (id) from task
+SELECT COUNT(id) from task
+ where due_date is NULL
+
+
+SELECT task.id, title, description, created, updated,due_date, user_id, status.name
+from task
+JOIN status ON task.status_id = status.id 
+where status.name= 'Done' 
+
+SELECT task.id, title, description, created, updated,due_date, user_id, status.name
+from task
+JOIN status ON task.status_id = status.id 
+where status.name != 'Done' 
+
+SELECT * from task
+ORDER BY(created)
+
+SELECT * from task
+ORDER BY(created)
+LIMIT 1
+
+select * from task
+WHERE description is not NULL and due_date is not NULL
```

Comment:
Here you were supposed to find tasks that **contain** the word _database_ in the `title` or `description`. You have to use a specific operand. 😉